packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -68,28 +72,44 @@ export class AtomicSearchBox {
    */
   @Prop() public numberOfQueries = 8;
 
+  /**
+   * Defining this option makes the search box standalone.
+   *
+   * This option defines the default URL the user should be redirected to, when a query is submitted.
+   * If a query pipeline redirect is triggered, it will redirect to that URL instead
+   * (see [query pipeline triggers](https://docs.coveo.com/en/1458)).
+   */
+  @Prop() public redirectionUrl?: string;
+
   @AriaLiveRegion('search-box')
   protected ariaMessage!: string;
 
   public initialize() {
     this.id = randomID('atomic-search-box-');
     this.querySetActions = loadQuerySetActions(this.bindings.engine);
-    this.searchBox = buildSearchBox(this.bindings.engine, {
-      options: {
-        id: this.id,
-        numberOfSuggestions: 0,
-        highlightOptions: {
-          notMatchDelimiters: {
-            open: '<span class="font-bold">',
-            close: '</span>',
-          },
-          correctionDelimiters: {
-            open: '<span class="font-normal">',
-            close: '</span>',
-          },
+
+    const searchBoxOptions = {
```

Comment:
One problem with this is for SinglePageApp. 

Users might want to have the same component on their home page as their full search page (eg: in the header of the website), but changing (adding/removing) the `redirectionUrl` property depending on which page is currently displayed, so that the search box behave differently.

By using the `initialize()` function I think we forfeit that possibility ?
Could we add a `@Watch()` decorator, which rebuild the controller ?

Reply:
✅